oop/music/classical.py

```diff
@@ -0,0 +1,12 @@
+from oop.music.beats import Beats
+
+class Classical(Beats):
+    """ a class that identifies Classical music """
+    def __init__(self, title: str, bpm: int, instruments: [], no_lyrics: bool, brass: bool, brass_instrument: str):
+        super().__init__(title, bpm, instruments, no_lyrics)
+        self.brass = brass
```

Comment:
similar issue here - if brass is always True there's no need to require it in the __init__() constructor.